blockx/assetlist.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "blockx-atlantis-testnet",
```

Comment:
the chain_name and directory name must match and cannot have dashes. I'd think this this should be relocated into the /testnets/ directory, and then the new directory and chain_name be blockxtestnet

Reply:
Huge thanks for detailed guidance & direction. Apologies for not noticing the `/testnets/` directory though.

Done. Relocated & Renamed. Matched.

Reply:
The same change is require on the chain.json file. Currently the chain_name property still doesn't match.

Reply:
Fixed with thanks.

Apologies for not taking care of this too.

Highly appreciated.

Kindly find the fix in the last commit #[151314c](https://github.com/cosmos/chain-registry/pull/4758/commits/151314c35fdb929655b68ae03b591971f7d5bc51)